src/seedu/addressbook/AddressBook.java

```diff
@@ -200,7 +200,18 @@
      */
     public static void main(String[] args) {
         showWelcomeMessage();
```

Comment:
Think about other constructs such as variables that also contribute to abstraction.